reflex/utils/format.py

```diff
@@ -308,6 +309,8 @@ def format_prop(
             # Dump the prop as JSON.
             prop = json_dumps(prop)
     except TypeError as e:
+        if "The style prop" in str(e):
```

Comment:
String scanning exceptions is a code smell, because if the exception message gets changed or reworded, it's not obvious that this needs to be changed.

Consider defining a new exception like

```python
class InvalidStyleProp(TypeError):
    """An invalid object was passed for a style prop."""
```

And raise/catch that here.

Reply:
got it